test/test_sdo.py

```diff
@@ -2,8 +2,11 @@
 import unittest
 # import binascii
 import canopen
+from canopen.objectdictionary import ODVariable
+import canopen.objectdictionary.datatypes as dt
 
 EDS_PATH = os.path.join(os.path.dirname(__file__), 'sample.eds')
+DATAEDS_PATH = os.path.join(os.path.dirname(__file__), 'all_datatypes.eds')
```

Comment:
Why not simply integrate the data types test objects into the existing sample?

Reply:
Separation of concerns. I didn't want to taint the existing test cases that sample uses.

Reply:
Alright.